realm/realm-library/src/objectServer/java/io/realm/SyncManager.java

```diff
@@ -230,6 +232,25 @@ private static synchronized void removeSession(SyncConfiguration syncConfigurati
         }
     }
 
+    /**
+     * Retruns the all valid sessions belonging to the user.
```

Comment:
@realm/java If we are sure that we won't use `ObjectStore`'s `all_sessions()` in the future, i am fine with this implementation. But if we do plan to migrate more things which might have impact on this feature, i suggest we postpone this ... According to my experience, all features we implemented in java first then move to Object Store is quite painful ... It would be a huge waste of time later ... (I am struggling with the last step of integration Object Store schema now ...)